### PR TITLE
Nozomi fix broken proxy2

### PR DESCRIPTION
**Pull request by nicoloereni**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [x] Tests
- [x] Documentation 

Screenshot 2024-12-18 at 11 24 04 AM
4c)

### CODE REVIEW

File: Packs/NozomiNetworks/ReleaseNotes/1_0_13.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Nozomi Networks
+
+- Fix proxy feature
```
Comment (YairGlik) [inline Packs/NozomiNetworks/ReleaseNotes/1_0_13.md:6]:
```suggestion
Fix proxy feature.
```

**Review by YairGlik — CHANGES_REQUESTED** [review]
Good job!
The package version in `pack_metadata.json` needs to be updated.
Please run the following command to update the release notes file:
`demisto-sdk update-release-notes`

**Comment by YairGlik** [comment]
Hi @nicoloereni,
The package version in `pack_metadata.json` needs to be updated.

**Comment by nicoloereni** [comment]
https://github.com/user-attachments/assets/cfca8e06-8bd9-4e03-a22e-3ee0b985b2d9

**Review by YairGlik — APPROVED** [review]

**Pull request merged**